ts/main.ts

```diff
@@ -0,0 +1,64 @@
+// interface for Monster
+interface Monster {
+  id: number;
+  name: string;
+  type: string;
+  description: string;
+  locations: [];
+}
+
+const $row = document.querySelector('.row');
+
+async function makeMonsters(): Promise<void> {
+  if (!$row) throw new Error('Could not load row');
+  for (let i = 1; i < 59; i++) {
+    try {
+      const response = await fetch(`https://mhw-db.com/monsters/${i}`);
+      if (!response.ok) {
+        throw new Error('This is not ok');
+      }
+      const monster = (await response.json()) as Monster;
+      console.log(monster);
+      const newPage = renderMonster(monster);
+      $row.append(newPage);
+    } catch (error) {
+      console.log('Error:', error);
+    }
+  }
+}
```

Comment:
Shouldn't we be using the main monster fetch? You're looping through and making 58 API calls here. Can't you just use the `'https://mhw-db.com/monsters'` endpoint and get the overall list of monsters instead of going through each?